pennylane/labs/resource_estimation/templates/subroutines.py

```diff
@@ -56,3 +56,7 @@ def resource_params(self) -> dict:
     def resource_rep(cls, num_wires) -> CompressedResourceOp:
         params = {"num_wires": num_wires}
         return CompressedResourceOp(cls, params)
+
+    @staticmethod
+    def tracking_name(num_wires) -> str:
+        return f"QFT({num_wires})"
```

Comment:
We don't need this! We can make another modification to the `_clean_gate_counts` function in `resources_tracking.py` to separately count gates that act on different number of wires / have different parameters. 

For the purpose of filtering via the gate set, we should just use the base implementation of the tracking_name method.

```suggestion
```